ops/check.sh

```diff
@@ -103,6 +103,7 @@ rgba_check() {
     --exclude-dir='node_modules' \
     --exclude-dir='coverage' \
     --exclude='*.html' \
+    --exclude='index.tsx' \
```

Comment:
revert it! 
`No rgba` is our convention.

Reply:
done